### PR TITLE
Renaming "Experience" to "Appearance"

### PR DESCRIPTION
**Pull request by GsnMithra**
I hope you're well. I'm suggesting a minor change: renaming the "Experience" tab to "Appearance." This aims for a more standard term, enhancing clarity.

Your thoughts on this adjustment would be greatly appreciated. Thank you for your time.

Before: 
Before

After:
After

### CODE REVIEW

**Comment by charlesBochet** [comment]
@Bonapara

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (charlesBochet) [inline .idea/workspace.xml:1]: @GsnMithra could you remove this file and add it the .gitignore?

**Comment by charlesBochet** [comment]
@GsnMithra We have discussed with @Bonapara and we think that this change is a good idea as long as the "Experience" section does not propose further customizations that would not only be tied to the "Appearance"

I've made a comment about idea file. Once fixed, we can merge it, thank you!

**Comment by FelixMalfait** [comment]
Thanks for the proposal @GsnMithra!
The reasoning was that some more generic items would fit into that category (e.g. what page do you open by default when you open the app, what default timezone, etc.). If we were to rename I'd rather go with something even more generic (e.g. "Preferences") rather than something more specific, to avoid future dev work. This is something we will necessarily refactor, and moving from Appearance to Experience pushes that refactor closer in time.

File: .idea/workspace.xml (hunk 1 above)
Comment (GsnMithra) [reply .idea/workspace.xml]: Hey, thanks for replying. Yea I have removed the file and also added it to .gitignore with the upcoming commits.

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Pull request merged**